extension/src/telemetry/index.ts

```diff
@@ -62,7 +62,12 @@ const sanitizeProperties = (
     if (value === undefined || value === null) {
       continue
     }
-    sanitizeProperty(eventName as string, sanitizedProperties, key, value)
```

Comment:
[F] Not really sure why removing `EventName.EXPERIMENTS_RUNNER_COMPLETED` caused `eslint` to freak out here.